File: java/Node.java
public class Node {
    private Student mahasiswa;
    private Node next;

    Node() {
        this.mahasiswa=new Student();
        this.next=null;
    }

    Node(Student student) {
        this.mahasiswa=student;
        this.next=null;
    }

    public void setMahasiswa(Student mahasiswa) {
        this.mahasiswa = mahasiswa;
    }

    public void setNext(Node next) {
        this.next = next;
    }

    public Student getMahasiswa() {
        return mahasiswa;
    }
    public Node getNext() {
        return next;
    }
}

File: py/Node.py
from Student import Student

class Node :
    def __init__(self) :
        self.__mahasiswa=Student()
        self.__next=None

    def setMahasiswa(self,mahasiswa) : self.__mahasiswa=mahasiswa
    def setNext(self,next) : self.__next=next

    def getMahasiswa(self) : return self.__mahasiswa
    def getNext(self) : return self.__next